deps/+CGAL/CGAL.cmake

```diff
@@ -3,8 +3,8 @@ add_cmake_project(
     # GIT_REPOSITORY https://github.com/CGAL/cgal.git
     # GIT_TAG        bec70a6d52d8aacb0b3d82a7b4edc3caa899184b # releases/CGAL-5.0
     # For whatever reason, this keeps downloading forever (repeats downloads if finished)
-    URL      https://github.com/CGAL/cgal/archive/refs/tags/v5.4.zip
-    URL_HASH SHA256=d7605e0a5a5ca17da7547592f6f6e4a59430a0bc861948974254d0de43eab4c0
+    URL      https://github.com/CGAL/cgal/archive/refs/tags/v6.0.1.zip
+    URL_HASH SHA256=6aa3837ebcefc39a53a7e6ac8ac08d7695d942e2eaab3709dc43da118cd10bc4
 )
```

Comment:
I think also libigl should be updated as it uses deprecated cgal header files.

Reply:
That probably makes sense. I just went with a minimal set of updates to make things work with a modern set of libraries as found in a contemporary system. I just didn't immediately want to open a can of worms since there are lots of other libraries from which deprecated headers are used, like curl.